app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -324,5 +324,10 @@
 
   <dimen name="home_fragment_padding_top">48dp</dimen>
   <dimen name="home_fragment_padding_bottom">100dp</dimen>
-  
+
+  <dimen name="promoted_story_card_layout_width">280dp</dimen>
```

Comment:
This can be removed as its same across all files. You can keep this only in `values/dimens.xml` fie

Reply:
done